lua/packer/plugin_utils.lua

```diff
@@ -174,10 +174,8 @@ plugin_utils.post_update_hook = function(plugin, disp)
             stdout = jobs.logging_callback(hook_output.err, hook_output.output, nil, disp,
                                            plugin_name)
           }
-          local cmd = {
-            os.getenv('SHELL'), '-c', 'cd ' .. plugin.install_path .. ' && ' .. plugin.run
-          }
-          return await(jobs.run(cmd, {capture_output = hook_callbacks})):map_err(
+          local cmd = {os.getenv('SHELL') or vim.o.shell, '-c', plugin.run}
```

Comment:
shall we use vim.o.shell as default?

Reply:
Ah, right, that does look a bit weird. My reluctance to use `vim.o.shell` by default comes from the fact that the option can contain spaces and quoted arguments, so it'd have to be tokenized properly so it can be split (`:help shell-unquoting` has more info on this). Neovim has an internal function to parse this option (`shell_build_argv()`) but I thought it was a bit overkill to try and make a Lua version of it, at least in this PR. Right now `'shell'` is used as a fallback when there is no `$SHELL` environment variable (the test actually failed on GitHub CI for that very reason)